eng/pipelines/code-quality-reports.yml

```diff
@@ -118,29 +118,18 @@ jobs:
         displayName: 'Generate Linting Reports'
         condition: and(always(), ne(variables['LintingGoals'], ''))
 
-      - pwsh: |
-          $artifactZip = "$(Build.ArtifactStagingDirectory)/linting-reports.zip"
-          $destinationDir = "$(Build.ArtifactStagingDirectory)/zipped-linting-reports"
-          if (Test-Path $artifactZip) {
-            New-Item -ItemType Directory -Path "$destinationDir"
-            Move-Item -Path $artifactZip -Destination "$destinationDir"
-          }
-        displayName: Move linting report zip to individual directory for artifact upload
-        condition: succeededOrFailed()
-
-      - template: /eng/common/pipelines/templates/steps/publish-artifact.yml
-        parameters:
-          DisplayName: 'Publish Linting Reports'
-          ArtifactPath: '$(Build.ArtifactStagingDirectory)/zipped-linting-reports'
-          ArtifactName: 'linting-reports'
+      - task: PublishPipelineArtifact@1
```

Comment:
After the 1ES work you can no longer publish an artifact with this task. You would need to use https://github.com/Azure/azure-sdk-for-java/blob/main/eng/common/pipelines/templates/steps/publish-1es-artifact.yml

Reply:
You can also pass in a custom condition if needed.